nanover-rs/src/tracked_simulation/openmm.rs

```diff
@@ -233,3 +242,20 @@ fn add_force_map_to_frame(force_map: &CoordMap, frame: &mut FrameData) {
         .insert_float_array("forces.user.sparse", sparse)
         .unwrap();
 }
+
+fn compute_potential_energy_correction(force_map: &CoordMap, simulation: &OpenMMSimulation) -> f64 {
```

Comment:
May be useful to have a comment here outlining that this function unfortunately has to retrieve the positions from openmm independently of what is already happening for the frame, and that ideally we will find some way to avoid it in the future. Maybe it makes more sense to raise that as an issue as well after the PR is merged, but a comment seems good too so it doesn't get lost or forgotten along the way.

Reply:
I have added a comment and a TODO about this. I agree, putting it into an issue as well will be useful.